Packages/vcs/Lib/configurator.py

```diff
@@ -258,81 +199,121 @@ def detach(self):
 
         # if all of the widgets have been cleaned up correctly, this will delete the manager
         vtk_ui.manager.delete_manager(self.interactor)
-        self.interactor.GetRenderWindow().Render()
+        self.render_window.Render()
 
     def release(self, object, event):
         if self.clicking is None:
             return
 
         if datetime.datetime.now() - self.clicking[1] < datetime.timedelta(0, .5):
-
             point = self.clicking[0]
             self.clicking = None
+
             if self.creating:
                 self.click_locations.append(point)
                 if len(self.click_locations) == CLICKS_TO_CREATE[self.creating]:
                     self.create()
                 return
 
-            if self.target and self.shift() == False and self.target.handle_click(point):
+            if self.target and self.shift() is False and self.target.handle_click(point):
                 return
 
             if self.shift() and type(self.target) != editors.group.GroupEditor:
                 self.target = editors.group.GroupEditor(self.interactor, (self.target,))
 
-            clicked = None
```

Comment:
Moved all of the click determination logic to actor_at_point, which uses a `vtkPropPicker` to retrieve the topmost actor at a point.